scripts/enrich_rulings_ai.py

```
@@ -14,94 +15,136 @@
+client = OpenAI(api_key=os.environ.get("OPENAI_API_KEY"))
+
+def call_openai_api(prompt: str, model: str = "gpt-4o") -> dict | None:
+    """
+    Calls the OpenAI API with a given prompt and returns the JSON response.
+    """
+    try:
+        response = client.chat.completions.create(
+            model=model,
+            messages=[
+                {"role": "system", "content": "You are a helpful assistant designed to output JSON."},
+                {"role": "user", "content": prompt},
```

Ruff (0.11.9)

Line too long (102 > 100)

(E501)